server/utils/utils.ts

```diff
@@ -141,6 +141,21 @@ export function isDateTimeRangeCurrent(startDateTimeStr: string, endDateTimeStr:
   return startDateTime <= dateToday && endDateTime >= dateToday
 }
 
+export function isDateTimeAfterCurrent(afterDateTimeStr: string) {
+  if (!afterDateTimeStr) {
+    return false
+  }
+
+  const dateToday = new Date()
```

Comment:
I'm not too familiar with dates in Javascript, so better to ask: should we not be using `Date.UTC` here and in the next line? The helm yaml files are expressed in UTC (maybe they should have the Z character to make that explicit?).